amounts.md

```diff
@@ -0,0 +1,341 @@
+There are 1 occurrences of 0.00232849 BTC output.
+There are 1 occurrences of 0.00250764 BTC output.
+There are 1 occurrences of 0.00251425 BTC output.
+There are 40 occurrences of 0.00262144 BTC output.
+There are 1 occurrences of 0.00319055 BTC output.
+There are 1 occurrences of 0.00501425 BTC output.
+There are 34 occurrences of 0.00524288 BTC output.
+There are 1 occurrences of 0.00628344 BTC output.
+There are 1 occurrences of 0.00766987 BTC output.
+There are 1 occurrences of 0.008003 BTC output.
+There are 1 occurrences of 0.00889539 BTC output.
+There are 31 occurrences of 0.01048576 BTC output.
+There are 1 occurrences of 0.01064874 BTC output.
+There are 1 occurrences of 0.01079982 BTC output.
+There are 1 occurrences of 0.01080556 BTC output.
+There are 1 occurrences of 0.01237489 BTC output.
+There are 1 occurrences of 0.01273657 BTC output.
+There are 1 occurrences of 0.01345343 BTC output.
+There are 1 occurrences of 0.0138582 BTC output.
+There are 1 occurrences of 0.01415412 BTC output.
+There are 1 occurrences of 0.0147032 BTC output.
+There are 1 occurrences of 0.01599117 BTC output.
+There are 1 occurrences of 0.01711393 BTC output.
+There are 1 occurrences of 0.01849849 BTC output.
+There are 28 occurrences of 0.02097152 BTC output.
+There are 1 occurrences of 0.02611393 BTC output.
+There are 1 occurrences of 0.03132254 BTC output.
+There are 1 occurrences of 0.031777 BTC output.
+There are 1 occurrences of 0.0360725 BTC output.
+There are 1 occurrences of 0.0384578 BTC output.
+There are 19 occurrences of 0.04194304 BTC output.
+There are 1 occurrences of 0.05586343 BTC output.
+There are 1 occurrences of 0.06800123 BTC output.
+There are 1 occurrences of 0.06908003 BTC output.
+There are 12 occurrences of 0.08388608 BTC output.
+There are 1 occurrences of 0.10004366 BTC output.
+There are 1 occurrences of 0.10557624 BTC output.
+There are 1 occurrences of 0.10746535 BTC output.
+There are 1 occurrences of 0.11578316 BTC output.
+There are 1 occurrences of 0.13324562 BTC output.
+There are 1 occurrences of 0.1543588 BTC output.
+There are 7 occurrences of 0.16777216 BTC output.
+There are 4 occurrences of 0.33554432 BTC output.
+There are 2 occurrences of 0.67108864 BTC output.
+```
+
+#### Dust Strategy
+
+Our next thought was to make a drastic change on the amount organization structure, but we realized the nature of the change may depend on how fees are handled and that may depend on our dust handling strategy, thus it's reasonable to continue with that.  
+So far we've created a transaction that isn't standard, because many outputs, like one satoshi outputs are present. To manage this issue and make our transaction standard, we want to make sure users don't register small outputs by setting a dust limit.  
+Dismissing the dust by the user is fine, because it's not significant amount. Let's assume our dust limit is 1000 satoshis, which should satisfy the standardness rules.
```

Comment:
```suggestion
Dismissing the dust by the user is fine, because it's not significant amount. Let's assume our dust limit is 1023 satoshis, which should satisfy the standardness rules.
```

we don't want to have dust below the last standard amount, no?

not sure if my change should be `1023` or `1024`.

Reply:
I see this is discussed later in `Descending Powers of 2`.

But then it might be worth to mention here that we had small coins below a standard denomination, which was a problem. then later we tell how we fixed the problem.

Reply:
At this step the dust is 1000, we changed it later.